api/opencl_runtime_layer.asciidoc

```diff
@@ -3591,7 +3591,7 @@ Otherwise, it returns a `NULL` value with one of the following error values
 returned in _errcode_ret_:
 
   * {CL_INVALID_CONTEXT} if _context_ is not a valid context.
-  * {CL_INVALID_OPERATION} if no devices in _context_ support Pipes.
+  * {CL_INVALID_OPERATION} if no devices in _context_ support pipes.
```

Comment:
You changed capitalization here but not above

Reply:
Thanks - you are very observant!

I initially thought that the "feature names" were not capitalized in the main spec but were capitalized in the appendix, but it looks like there isn't a consistent capitalization policy.  For this PR, I've tried to be as consistent as possible with Pipes and Intermediate Language Programs, since both appear to be consistently capitalized, but to really "fix" this we'd want to take a pass through the specs to apply a consistent policy.  I'm willing to do this but I'd like to do it as a separate PR.